projects/1.18/assets/nuclearscience/nuclearscience/lang/zh_cn.json

```
@@ -0,0 +1,132 @@
+  "item.nuclearscience.frequencycard" : "频率卡",
+  "block.nuclearscience.gascentrifuge" : "气体离心机",
+  "block.nuclearscience.nuclearboiler" : "核能锅炉",
+  "block.nuclearscience.chemicalextractor" : "化学提取机",
+  "block.nuclearscience.radioisotopegenerator" : "放射性同位素温差发电机",
+  "block.nuclearscience.freezeplug" : "防冻塞",
+  "block.nuclearscience.turbine" : "蒸汽涡轮机",
+  "block.nuclearscience.reactorcore" : "裂变反应堆芯",
+  "block.nuclearscience.electromagnet" : "电磁铁",
+  "block.nuclearscience.electromagneticglass" : "电磁玻璃",
+  "block.nuclearscience.electromagneticbooster" : "电磁推进器",
+  "block.nuclearscience.electromagneticswitch" : "电磁开关",
+  "block.nuclearscience.fusionreactorcore" : "聚变反应堆芯",
+  "block.nuclearscience.fuelreprocessor" : "燃料再处理机",
+  "block.nuclearscience.radioactiveprocessor" : "放射性处理机",
+  "block.nuclearscience.msrfuelpreprocessor" : "MSR 燃料预处理机",
+  "block.nuclearscience.plasma" : "等离子",
+  "block.nuclearscience.particleinjector" : "粒子注入器",
+  "block.nuclearscience.quantumcapacitor" : "量子电容",
+  "block.nuclearscience.teleporter" : "传送机",
+  "block.nuclearscience.controlrodassembly" : "控制棒组件",
+  "block.nuclearscience.blocklead" : "陶瓷-铅辐射屏蔽器",
+  "block.nuclearscience.msrreactorcore" : "MSR 反应堆芯",
+  "block.nuclearscience.heatexchanger" : "热交换机",
+  "block.nuclearscience.moltensaltpipevanadiumsteelceramic" : "钒钢-陶瓷熔融盐管道",
+  "block.nuclearscience.moltensaltsupplier" : "熔融盐供给机",
+  "block.nuclearscience.radioactivesoil" : "放射性泥土",
+  "block.nuclearscience.meltedreactor" : "过热熔化的反应堆",
+  "block.nuclearscience.siren" : "警报器",
+  "block.nuclearscience.atomicassembler" : "原子装配机",
+  "container.gascentrifuge" : "气体离心机",
+  "container.nuclearboiler" : "核能锅炉",
+  "container.chemicalextractor" : "化学提取机",
+  "container.radioisotopegenerator" : "放射性同位素温差发电机",
+  "container.reactorcore" : "裂变反应堆芯",
+  "container.particleinjector" : "粒子注入器",
+  "container.quantumcapacitor" : "量子电容",
+  "container.fuelreprocessor" : "燃料再处理机",
+  "container.radioactiveprocessor" : "放射性处理机",
+  "container.msrfuelpreprocessor" : "MSR 燃料预处理机",
+  "container.freezeplug" : "防冻塞",
+  "container.msrreactorcore" : "MSR 反应堆芯",
+  "container.moltensaltsupplier" : "熔融盐供给机",
+  "container.atomicassembler" : "原子装配机",
+  "gui.machine.usage" : "功耗：%s",
+  "gui.machine.voltage" : "电压：%s",
+  "gui.machine.output" : "输出：%s",
+  "gui.machine.current" : "电流：%s",
+  "gui.machine.transfer" : "输出功率：%s",
+  "gui.machine.stored" : "电量：%s",
+  "gui.particleinjector.charge" : "电量：%s",
+  "gui.particleinjector.matter" : "物质",
+  "gui.particleinjector.cells" : "单元",
+  "gui.reactorcore.deuterium" : "氘",
+  "gui.reactorcore.temperature" : "温度：%s",
+  "gui.reactorcore.warning" : "过 热！",
+  "gui.msrreactorcore.fuel" : "燃料量（mB）：%s",
+  "gui.msrreactorcore.nofreezeplug" : "未找到防冻塞！",
+  "itemGroup.itemgroupnuclearscience" : "核科学",
+  "death.attack.radiation" : "%s刚刚速通了进化论",
+  "death.attack.plasma" : "%s被电离了",
+  "effect.nuclearscience.radiation" : "辐射",
+  "message.geigercounter.text" : "每小时 %s Rad",
```

```suggestion
  "message.geigercounter.text" : "每小时 %s 拉德",
```
[参考](https://www.termonline.cn/search?k=rad&r=1645276166351)。

---

我在其他地方使用的单位均为英文缩写，所以这个地方我觉得还是该用 Rad，不过有点不太好就是数学领域也有一个意义完全不同的 Rad 单位。

---

就是为了消歧义用中文

---

Okay